locales/en/index.yml

```diff
@@ -389,19 +389,19 @@ profile:
             description:
               one: "IO collects only the information "
               two: "necessary "
-              three: "to monitor the performance of the services provided to you"
+              three: "to monitor the performance of the services provided to you."
             cta: "Why?"
           security:
             description:
-              one: "We take security measures to protect your data"
+              one: "We take security measures to protect your data."
             cta: "Learn more"
           gdpr:
             description:
               one: "The IO team works with other providers in accordance with "
-              two: "GDPR regulations"
+              two: "GDPR regulations."
             cta: "See our providers"
           additionalInformation:
-            description: "For more information, please read"
+            description: "For more information, please read "
```

Comment:
```suggestion
            description: "For more information, please read"
```
I guess it is a typing error

Reply:
I actually added it on purpose (but I missed it in the list of changes sorry 😅). Without the space the current behaviour is:
<img width="336" alt="image" src="https://user-images.githubusercontent.com/5963683/153250528-cb573c07-3604-4595-a59a-30ee77b84460.png">
That last space is present in the italian version too.